packages/proxyIdentity/contracts/IdentityManager.sol

```diff
@@ -1,12 +1,40 @@
-pragma solidity 0.8.0;
+pragma solidity 0.4.26;
 
+import "@optionality.io/clone-factory/contracts/CloneFactory.sol";
+import "openzeppelin-solidity/contracts/introspection/ERC165Checker.sol";
```

Comment:
Looks like you're using this `CloneFactory` from optionality.io. I'm not two familiar with it. Why use it instead of the Clones library provided by OpenZeppelin? https://docs.openzeppelin.com/contracts/4.x/api/proxy#Clones

Reply:
Didn't know they added it. After looking at discussion https://github.com/OpenZeppelin/openzeppelin-contracts/issues/1162 I decided they are not going to add it in near future. I will try to use it